tools/i18n/update-for-legacy-translations

```diff
@@ -158,7 +158,7 @@ LEGACY_STRINGS_MAP = {
     "Who can create web-public channels": "Who can create web-public streams",
     "Who can move messages to another channel": "Who can move messages to another stream",
     "Who can post to this channel": "Who can post to the stream?",
-    "Who can unsubscribe others from this channel?": "Who can unsubscribe others from this stream?",
+    "Who can unsubscribe others from this channel": "Who can unsubscribe others from this stream?",
```

Comment:
I think you wanted to remove this from both sides.

Reply:
Yes, removed at https://github.com/zulip/zulip/compare/6b7e7f35f42bbeb4732df821d4fd3f16f838b63c..7ecebc6ff99d9f9dc0d5c96f5ba73eac2d2f0af2